src/newrelic/harvest.go

```diff
@@ -27,6 +27,7 @@ type Harvest struct {
 	CustomEvents      *CustomEvents
 	ErrorEvents       *ErrorEvents
 	SpanEvents        *SpanEvents
+	LogEvents         *LogEvents
```

Comment:
Shouldn't `func NewHarvest()` be updated to account for this new field?

Reply:
Yes that needs to happen. But I think it should be changed with the other changes adding functionality to the daemon. This change is just meant to add the data structure. If I added it to NewHarvest(), I'd also need to pipe through the limits for hl.LogEventConfig.Limit to be properly called.

Reply:
Ok. Makes sense.